edx_notifications/channels/tests/test_urban_airship.py

```diff
@@ -72,3 +72,17 @@ def test_single_user_notification(self, mock_ua_push_api):
         mock_ua_push_api.return_value = {'ok': 'true'}
         response = publish_notification_to_user(10, self.msg, preferred_channel='urban-airship')
         self.assertEqual(response['ok'], 'true')
+
+    @patch("edx_notifications.channels.urban_airship.UrbanAirshipNotificationChannelProvider.call_ua_push_api")
+    def test_tag_group_announcement(self, mock_ua_push_api):
+        """
+        Test publish notification to a tag group
```

Comment:
nit: docstring could be `Test publish notification to a tag group when notification_type is present`